rectools/metrics/auc.py

```diff
@@ -513,12 +535,28 @@ def calc_auc_metrics(
     """
     results = {}
 
-    k_max = max(metric.k for metric in metrics.values())
     insufficient_handling_needed = any(
-        metric.insufficient_handling != InsufficientHandling.IGNORE for metric in metrics.values()
+        metric.insufficient_handling != InsufficientHandling.IGNORE
+        for metric in metrics.values()
+        if metric.debias_config is None
```

Comment:
Why do we check only metrics without debias, but then use `insufficient_handling_needed` for all metrics, including that ones with debias?